modules/rational-use-package.el

```diff
@@ -4,8 +4,15 @@
 
 (require 'package)
 
-(setq package-archives '(("melpa" . "https://melpa.org/packages/")
-                         ("elpa" . "https://elpa.gnu.org/packages/")))
+;; gnu and nongnu is needed for packages like Org-mode
+(setq package-archives
```

Comment:
I'd update this to:

```emacs-lisp
(setq package-archives '(("gnu" . "https://elpa.gnu.org/packages/")
                         ("nongnu" . "https://elpa.nongnu.org/nongnu/")
                         ("melpa" . "https://melpa.org/packages/")))
```

Reply:
Another thought, the "gnu" repo is default in 27.x, while the "gnu" and "nongnu" are default in 28.x, so, only need to add "nongnu" if emacs version is 27.x and then always add "melpa".  With that in mind, the `add-to-list` approach allows adding a check on emacs version around the "nongnu" repo. Thus:

    (when (< (string-to-number emacs-version) 28)
      (add-to-list 'package-archives (cons "nongnu" "https://elpa.nongnu.org/nongnu/")) 'append)
    (add-to-list 'package-archives (cons "melpa" "https://melpa.org/packages/") 'append)